allure-report-face/src/components/widgets-grid/WidgetsGridView.js

```diff
@@ -7,7 +7,9 @@ import {className} from '../../decorators';
 import allurePlugins from '../../pluginApi';
 
 const widgetTpl = (id) => `<div class="widget island" data-id="${id}">
-    <div class="widget__handle fa fa-arrows"></div>
+    <div class="widget__handle">
+        <span class="fa fa-ellipsis-v"></span> <span class="fa fa-ellipsis-v"></span>
```

Comment:
@lanwen I am 100% sure that your icon also can't help them. Now it is 6 number on dice.

![image](https://cloud.githubusercontent.com/assets/812240/15567038/611f1720-2324-11e6-90fb-feeda262cabc.png)

If you would like to do it, you can do dotted background along whole side, like gmail does:

![image](https://cloud.githubusercontent.com/assets/812240/15567150/f7e042ec-2324-11e6-8dea-c7c2e88ae5c0.png)

Also, you can check out Google search, there is a [parity](https://www.google.ru/search?q=draggable+area&newwindow=1&hl=ru&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiU2d60nPfMAhUB1xoKHTScC-QQ_AUIBygB&biw=1185&bih=633#newwindow=1&hl=ru&tbm=isch&q=draggable+area+icon) between arrows and dots as draggable areas